architecture/aws/observability.md

```diff
@@ -5,61 +5,62 @@ reviewed: 2024-03-14
+- Customizable dashboards with automated alarms and remediating actions
 - Real-time telemetry
 
 :heavy_minus_sign: Cons:
 
-- Amazon CloudWatch does not support traces
-- AWS CloudWatch offers a [free tier](https://aws.amazon.com/cloudwatch/pricing/), but costs may escalate with increases usage, requiring dedicated monitoring
+- Does not support traces
+- [Costs](https://aws.amazon.com/cloudwatch/pricing/) can escalate with increased usage
 
 The Particular Service Platform collects metrics in two forms:
 
-- OpenTelemetry-based metrics, which can be collected by enabling OpenTelemetry and exporting the metrics to Amazon CloudWatch
-- Custom metrics with NServiceBus.Metrics which can be exported to Amazon CloudWatch.
+1. OpenTelemetry-based metrics - can be collected by [enabling OpenTelemetry](/nservicebus/operations/opentelemetry.md) and exporting the metrics to Amazon CloudWatch.
+1. Custom metrics with `NServiceBus.Metrics` - can be exported to Amazon CloudWatch - TODO: Link this to TODO2 below.
 
-Try the Amazon CloudWatch sample for NServiceBus metrics and logs →
+Try the Amazon CloudWatch sample for OpenTelemetry → TODO: Link this to TODO1 below.
 
 **TODO**:
 
-1. Create a sample that uses NSB metrics, which integrates with Cloudwatch
+1. Create a sample that uses Otel and integrates with CloudWatch
+1. Create a sample that uses NSB metrics, which integrates with CloudWatch
     1. Collect NSB metrics, send to cloudwatch
     2. Collect [NLog logs with extensions.logging](/samples/logging/extensions-logging/), [send to Cloudwatch](https://docs.aws.amazon.com/prescriptive-guidance/latest/patterns/configure-logging-for-net-applications-in-amazon-cloudwatch-logs-by-using-nlog.html)
 
-#### Amazon X-Ray
+## Amazon X-Ray
 
-[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can collect trace data from the applications, providing insights that can help identify issues or bottlenecks that could benefit from optimization.
+[Amazon X-Ray](https://aws.amazon.com/xray/) is a service that can receive trace data from applications. The trace data can provide insights that can help identify performance bottlenecks, edge case errors, and other hard to detect issues.
 
 :heavy_plus_sign: Pros:
 
-- A single tool to visualize application-level traces and infrastructure-level traces
-- AWS X-Ray creates a service map using trace data
-- Integrates with other AWS services, such as AWS Lambda, Amazon EC2, Amazon ECS and AWS Elastic Beanstalk
+- Single tool to visualize application-level and infrastructure-level traces
+- Creates a service map using trace data
+- Tightly integrated with other AWS services
 - Fully managed service
 
 :heavy_minus_sign: Cons:
 
-- AWS X-Ray offers a [free tier](https://aws.amazon.com/xray/pricing/), but costs may escalate with increases usage, requiring dedicated monitoring
-- Pricing is based on the amount and the type of telemetry collected
-- AWS X-Ray is designed to work within the AWS ecosystem
```

Comment:
I think they are different things.
1) It's great that this works well with other AWS services, if this would not have been true, then it would be listed as a con
2) There may be challenges when using this with tools outside the Amazon stack, because it was primarily built to be used with other AWS services. This is, for me, a con because the opposite pro would have been, "This integrates well with non-AWS services." 

Therefore, "integrate with AWS services" and "integrate with non-aws services" are just 2 different "attributes" that are weighed separately.
Does that make sense?

Reply:
While it makes sense in general, I think it makes less sense for the target audience for this section of our docs.

I posit that the audience of the "AWS Architecture" section on our docs are people looking for guidance on how to build a system using the AWS ecosystem, whereas people that want to mix-and-match ecosystems are more likely to be in one of the specific technology pages rather than the architecture section.

Reply:
Ok, that's fair, resolving then